.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Release on PyPI
+
+on:
+  push:
+    branches:
+    - '*'
```

Comment:
Why do we have branches `*` here?

I don't think it should be here.  For example, sometimes people do push development branches onto the main repo because they're too lazy to create a fork, and this should not trigger a pypi release.  I don't think there should be any `branches` here at all.

Reply:
Corrected.